open_samus_returns_rando/custom_pickups.py

```diff
@@ -0,0 +1,30 @@
+from open_samus_returns_rando.patcher_editor import PatcherEditor
+
+
+def _get_actor(editor: PatcherEditor):
+    actor_to_copy = {
+        "scenario": "s000_surface",
+        "layer": "9",
+        "actor": "LE_PowerUP_Morphball",
+    }
+    new_actor = editor.resolve_actor_reference(actor_to_copy)
+    return new_actor
```

Comment:
I just realised that the naming is confusing. This is not a new actor, this is just the actor you use as a reference.